src/utilities/__tests__/buildASTSchema-test.js

```diff
@@ -285,6 +285,12 @@ describe('Schema Builder', () => {
     const sdl = dedent`
       interface EmptyInterface
     `;
+
+    const definition = parse(sdl).definitions[0];
+    expect(
+      definition.kind === 'InterfaceTypeDefinition' && definition.interfaces,
+    ).to.deep.equal([], 'The interfaces property must be an empty array.');
+
```

Comment:
@mike-marcacci Why do you need this check?
This should be checked in parser tests.

Reply:
Adding this test was requested by @mjmahone [in this comment](https://github.com/graphql/graphql-js/pull/2084#discussion_r313964490).

Reply:
So Matt's comment was about testing if `buildSchema` supports both empty array and undefined:
> Do we have an integration test (probably not under the parser, but in buildASTSchema) for an AST having interfaces as undefined?

In its current form, you just duplicating parser tests since it's just checking `interfaces` value is an array and Matt suggested to modify it to `undefined` and test that result is the same.

At the moment we don't have any such test so think it should be a generic solution so I created #2203

Reply:
That's a great point; I assume I should just leave this as is and we can remove this test in a separate PR that fixes #2203?